frontend/src/components/articles/ArticleItem.vue

```diff
@@ -7,17 +7,28 @@ import InputNumber from "primevue/inputnumber";
 import InputText from "primevue/inputtext";
 import Textarea from "primevue/textarea";
 import { ref } from "vue";
+import { useRouter } from "vue-router";
 
+import { deleteArticle, updateArticle } from "@/backendClient";
 import { ArticleKind } from "@/global/types/appTypes";
 import type { Article } from "@/global/types/entities";
 
 const props = defineProps<{
   article: Article;
 }>();
+const router = useRouter();
 
 const articlesType = Object.values(ArticleKind);
 
 const editableArticle = ref(props.article);
+
+const onUpdateArticle = async () => {
+  await updateArticle(`${editableArticle.value.id}`, editableArticle.value);
+};
```

Comment:
Can we add a notification or toast that shows when saving data is successful? Currently, it works but I imagine the user not knowing if the saving worked and/or the click was registered.

Especially considering the instability and slowness of the current application the client will likely want some signal to know it worked.